templates/pages/setup/general/general_setup.html.twig

```diff
@@ -0,0 +1,337 @@
+        'helpdesk_doc_url',
+        config['helpdesk_doc_url'],
+        __('Simplified interface help link'),
+        field_options
+    ) }}
+
+    {{ fields.textField(
+        'central_doc_url',
+        config['central_doc_url'],
+        __('Standard interface help link'),
+        field_options
+    ) }}
+
+    {{ fields.numberField(
+        'decimal_number',
+        config['decimal_number'],
+        __('Default decimals limit'),
+        inline_field_options|merge({
+            'min': 1,
+            'max': 4,
+        })
+    ) }}
+
+    {{ fields.checkboxField(
+        'use_public_faq',
+        config['use_public_faq'],
+        __('Allow FAQ anonymous access'),
+        field_options
+    ) }}
+
+</div>
+
+<div class="hr-text">
+    <i class="ti ti-language"></i>
+    <span>{{ __('Translations') }}</span>
+</div>
+
+<div class="row ps-4">
+
+    {{ fields.checkboxField(
+        'translate_dropdowns',
+        config['translate_dropdowns'],
+        __("Translation of dropdowns"),
+        field_options
+    ) }}
+
+    {{ fields.checkboxField(
+        'translate_kb',
+        config['translate_kb'],
+        __("Knowledge base translation"),
+        field_options
+    ) }}
+
+    {{ fields.checkboxField(
+        'translate_reminders',
+        config['translate_reminders'],
+        __("Translation of reminders"),
+        field_options
+    ) }}
+
+</div>
+
+<div class="hr-text">
+    <i class="ti ti-list"></i>
+    <span>{{ __('Dynamic display') }}</span>
+</div>
+
+<div class="row ps-4">
+
+    {{ fields.numberField(
+        'dropdown_max',
+        config['dropdown_max'],
+        __('Page size for dropdown (paging using scroll)'),
+        inline_field_options|merge({
+            'min': 1,
+            'max': 200,
+        })
+    ) }}
+
+    {{ fields.dropdownNumberField(
+        'ajax_limit_count',
+        config['ajax_limit_count'],
+        __("Don't show search engine in dropdowns if the number of items is less than"),
+        inline_field_options|merge({
+            'min': 1,
+            'max': 200,
+            'toadd': [__('Never')],
+            'width': 'auto',
+        })
+    ) }}
+</div>
+
+<div class="hr-text">
+    <i class="ti ti-search"></i>
+    <span>{{ __('Search engine') }}</span>
+</div>
+
+<div class="row ps-4">
+
+    {{ fields.dropdownArrayField(
+        'allow_search_view',
+        config['allow_search_view'],
+        [
+            __('No'),
+            __('%1$s (%2$s)')|format(__('Yes'), __('last criterion')),
+            __('%1$s (%2$s)')|format(__('Yes'), __('default criterion'))
+        ],
+        __('Items seen'),
+        inline_field_options|merge({
+            'width': 'auto',
+        })
+    ) }}
+
+    {% set allow_search_view_options = inline_field_options %}
+    {% if not config['allow_search_view'] %}
+        {% set allow_search_view_options = allow_search_view_options|merge({
+            'readonly': true,
+            'disabled': true,
+        }) %}
```

Comment:
I know it was done like this previously, but why ? `Items seen` criteria and global search are disctinct features.

Reply:
I don't know, i'll remove the condition